www/language/fa-IR/fa-IR.site_menuitems.ini

```diff
@@ -23,7 +23,7 @@ MENU_DOWNLOADS_LATEST_METADESCRIPTION="آخرین نسخه جوملا را در
 MENU_DOWNLOADS_DOWNLOADS_PAGETITLE="دریافت جوملا! - همه نسخه های سیستم مدیریت محتوای متن باز جوملا!"
 MENU_DOWNLOADS_DOWNLOADS_PAGEHEADING="دریافت جوملا!"
 MENU_DOWNLOADS_DOWNLOADS_METADESC="Browse and download the Joomla! version you're looking for. All versions are available, even legacy versions."
-MENU_DOWNLOADS_EXTENSIONS_PAGETITLE="Download Official Joomla! Extensions: weblinks, install from web..."
+MENU_DOWNLOADS_EXTENSIONS_PAGETITLE="..."
```

Comment:
Also clearly wrong

Reply:
i've fixed this in crowdin. copy source.